core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -116,6 +116,7 @@ func NewEthBroadcaster(db *sqlx.DB, ethClient evmclient.Client, config Config, k
 			keystore: keystore,
 		},
 		estimator:        estimator,
+		resumeCallback:   resumeCallback,
```

Comment:
oh man, what a slip-up.

Any way to get a test on this?

Reply:
Done